templates/bake/Model/table.twig

```diff
@@ -14,13 +14,9 @@
  */
 #}
 {% set annotations = DocBlock.buildTableAnnotations(associations, associationInfo, behaviors, entity, namespace) %}
-<?php
-declare(strict_types=1);
-
-namespace {{ namespace }}\Model\Table;
-
-{% set uses = ['use Cake\\ORM\\Query;', 'use Cake\\ORM\\RulesChecker;', 'use Cake\\ORM\\Table;', 'use Cake\\Validation\\Validator;'] %}
-{{ uses|join('\n')|raw }}
+{% set generatedFunctions = ['initialize'] %}
+{% set classImports = fileBuilder.classImports(['Cake\\ORM\\Query', 'Cake\\ORM\\RulesChecker', 'Cake\\ORM\\Table', 'Cake\\Validation\\Validator']) %}
```

Comment:
This always add the imports even if the functions aren't generated below. However, this i show it was done previously.

Reply:
This is where I was talking about setting the imports in the command instead of the template.